module/token.js

```diff
@@ -0,0 +1,207 @@
+                template: "systems/cypher/templates/scene/tokenConfig.html"
+            });
+        }
+    });
+
+    TokenConfig.prototype.getData = (function () {
+        const superFunction = TokenConfig.prototype.getData;
+        return async function (options) {
+            let result = await superFunction.apply(this, arguments);
+            result.isPC = this.actor.data.type === "pc";
+            if (result.isPC) {
+                result.bar3 = this.object.getBarAttribute("bar3");
+            }
+
+            return result;
+        };
+    })();
+}
+
+/**
+ * Override to add a third attribute bar to the token
+ */
+function cypherTokenDrawAttributeBars() {
+    const bars = new PIXI.Container();
+    bars.bar1 = bars.addChild(new PIXI.Graphics());
+    bars.bar2 = bars.addChild(new PIXI.Graphics());
+    bars.bar3 = bars.addChild(new PIXI.Graphics());
+    return bars;
+}
+
+/**
+ * Override to add and draw a third attribute bar
+ */
+function cypherTokenDrawBars() {
+    if (!this.actor || (this.data.displayBars === CONST.TOKEN_DISPLAY_MODES.NONE)) return;
+    ["bar1", "bar2", "bar3"].forEach((b, i) => {
+        const bar = this.bars[b];
+        const attr = this.getBarAttribute(b);
+        if (!attr || (attr.type !== "bar")) return bar.visible = false;
+        this._drawBar(i, bar, attr);
+        bar.visible = true;
+    });
+}
+
+/**
+ * Override to update a third attribute bar
+ */
+function cypherOnUpdateBarAttributes(updateData) {
+    const update = ["bar1", "bar2", "bar3"].some(b => {
+      let bar = this.data[b];
+      return bar.attribute && hasProperty(updateData, "data."+bar.attribute);
+    });
+    if ( update ) this.drawBars();
+  }
+
+/**
+ * Override to change the way we draw bars, making them a bit more configurable
+ */
+function drawCypherBar(number, bar, data) {
+    const val = Number(data.value);
+    const pct = Math.clamped(val, 0, data.max) / data.max;
+    let h = Math.max((canvas.dimensions.size / 12), 8);
+    if (this.data.height >= 2) h *= 1.6;  // Enlarge the bar for large tokens
+
+    // Stacked bars, all atop one another
+    let yPositions = {
+        0: this.h - (3 * h),
+        1: this.h - (2 * h),
+        2: this.h - h
+    };
+
+    // Let's do at least one good thing by making these colors configurable
+    let colors = {
+        0: NUMENERA.attributeColors[0],
+        1: NUMENERA.attributeColors[1],
+        2: NUMENERA.attributeColors[2]
+    }
+    
+    let color = colors[number];
+
+    bar.clear()
+        .beginFill(0x000000, 0.5)
+        .lineStyle(2, 0x000000, 0.9)
+        .drawRoundedRect(0, 0, this.w, h, 3)
+        .beginFill(color, 0.8)
+        .lineStyle(1, 0x000000, 0.8)
+        .drawRoundedRect(1, 1, pct * (this.w - 2), h - 2, 2);
+
+    // Set position
+    let posY = yPositions[number];
+    bar.position.set(0, posY);
+}
+
+// Since the model for the Token config doesn't seem to want to cooperate, these stats are hard-coded to the 3 Cypher vitals
```

Comment:
Hardcoding for stats hich are so fundamental to the system doesn't seem like that big of a fault!

Reply:
Fair point! I was trying to think of some edge case where a PC might _not_ want to track these 3, and couldn't really come up with anything, so I'm not too beat up about it.